editor/filesystem_dock.cpp

```diff
@@ -76,6 +76,99 @@ Control *FileSystemList::make_custom_tooltip(const String &p_text) const {
 	return FileSystemDock::get_singleton()->create_tooltip_for_path(get_item_metadata(idx));
 }
 
+void FileSystemList::_line_editor_submit(String p_text) {
+	popup_editor->hide();
+
+	emit_signal(SNAME("item_edited"));
+	queue_redraw();
+}
+
+bool FileSystemList::edit_selected() {
+	int s = get_current();
+	ERR_FAIL_COND_V_MSG(!is_anything_selected(), false, "No item selected.");
+	ensure_current_is_visible();
+
+	Rect2 rect;
+	Rect2 popup_rect;
+	Vector2 ofs;
+
+	int icon_width = get_item_icon(s)->get_width();
+	int icon_height = get_item_icon(s)->get_height();
```

Comment:
Can be replaced with
```suggestion
	Vector2 icon_size = get_item_icon(s)->get_size();
```